stories/models/actor.py

```diff
@@ -0,0 +1,26 @@
+from django.utils.translation import ugettext_lazy as _
+from django.db import models
+
+
+class Actor(models.Model):
+    external_id = models.CharField(
+        verbose_name=_('ID'),
```

Comment:
Maybe 'External ID'? or 'Origin ID', this overlaps with the default PK verbose_name